### PR TITLE
This is the new einstein feature

### PR DESCRIPTION
**Pull request by qamaaar**
Hi, 

This new file contains the Einstein Analytics loader

### CODE REVIEW

**Comment by gozdy** [comment]
Hi @qamaaar , the PR you made was against master and not develop, once changing to develop there are conflicts to be solved. Please resolve them so the PR can be reviewed. Thanks!

File: src/main/resources/properties/secure/app-dev.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 auth:
   
-  orgId: "![yqKEWZ57M3XUasumfcwhVykLsfUM6yOEv2oEJ04vxZc=]"
+  orgId: "30370ff2-d147-43a4-9402-fb074b715423"
   
   # If auth mode = platform-credentials (ignored, otherwise):
-  username: "![Zjv8nFIUwDlayZ6O7DP31p1zQRfrwfjbkmi5DyNiD38=]"
-  password: "![yqKEWZ57M3XUasumfcwhV35QQl9B5mf5GBon8eY/x8o=]"
+  username: "qamar_"
+  password: "Mulesoftlondon2021!"
```
Comment (gozdy) [inline src/main/resources/properties/secure/app-dev.yaml:7]: Please remove personal credentials from the commit, in addition to modifying exposed credentials

**Review by richardmckinley — COMMENTED** [review]
Made some changes to streamline the implementation in line with other loaders. 
Added benefits metrics functionality. 
Tidied up properties, placing secure in correct location. 
Updated .gitignore

**Comment by richardmckinley** [comment]
Additional MUnits added for SFDC loader flows, maintaining coverage above 98% 
image

**Pull request merged**